docker-compose.yml

```diff
@@ -73,20 +76,6 @@ services:
       - "traefik.http.routers.${COMPOSE_PROJECT_NAME}Mailhog.rule=Host(`mailhog-${COMPOSE_DOMAIN}`)"
       - "traefik.http.services.${COMPOSE_PROJECT_NAME}Mailhog.loadbalancer.server.port=8025"
 
-  redis:
```

Comment:
I think we are using redis for symfony messenger: https://github.com/itk-dev/jira_economics/blob/develop/.env#L73
in the app: https://github.com/itk-dev/jira_economics/tree/develop/bundles/GraphicServiceOrder

Reply:
Added